osu.Game/Graphics/Containers/OsuHoverContainer.cs

```diff
@@ -17,7 +17,7 @@ public partial class OsuHoverContainer : OsuClickableContainer
 
         protected Color4 HoverColour;
 
-        protected Color4 IdleColour = Color4.White;
+        public Color4 IdleColour = Color4.White;
```

Comment:
If going to do this, better make sure that the change of it takes effect immediately when any consumer sets it. I don't see this happening in this diff.